OmniKit/MessageTransport/MessageBlocks/StatusResponse.swift

```diff
@@ -9,42 +9,6 @@
 import Foundation
 
 public struct StatusResponse : MessageBlock {
-    
-    public enum DeliveryStatus: UInt8, CustomStringConvertible {
-        case suspended = 0
-        case normal = 1
-        case tempBasalRunning = 2
-        case priming = 4
-        case bolusInProgress = 5
-        case bolusAndTempBasal = 6
-        
-        public var bolusing: Bool {
-            return self == .bolusInProgress || self == .bolusAndTempBasal
-        }
-        
-        public var tempBasalRunning: Bool {
-            return self == .tempBasalRunning || self == .bolusAndTempBasal
-        }
-
-        
-        public var description: String {
-            switch self {
-            case .suspended:
-                return LocalizedString("Suspended", comment: "Delivery status when insulin delivery is suspended")
-            case .normal:
-                return LocalizedString("Scheduled Basal", comment: "Delivery status when basal is running")
```

Comment:
The other DeliveryStatus, that will now be used, has this description as "Normal".  I think "Scheduled Basal" is better. Updating the enum name might be a good idea too.

Reply:
Good catch. Now fixed.